data/formats-data.js

```diff
@@ -5145,7 +5152,7 @@ exports.BattleFormatsData = {
 		tier: "NU",
 	},
 	vullaby: {
-		randomBattleMoves: ["knockoff", "roost", "taunt", "whirlwind", "toxic", "defog", "uturn", "bravebird"],
+		randomBattleMoves: ["knockoff", "roost", "heatwave", "mirrormove", "defog", "uturn", "bravebird"],
```

Comment:
+nasty plot, dark pulse, air slash 
(weak armor, np set)

Reply:
purposefully avoided this to prevent option overload that leads to bad things like darkpulse/heat wave/bravebird/uturn